asapdiscovery-alchemy/asapdiscovery/alchemy/cli.py

```diff
@@ -259,23 +266,44 @@ def gather(network: str, allow_missing: bool):
     default="planned_network.json",
     show_default=True,
 )
-def status(network: str):
+@click.option(
+    "--errors",
+    is_flag=True,
+    default=False,
+    help="Output errors from the network, if any.",
+)
+@click.option(
+    "--with-traceback",
+    is_flag=True,
+    default=False,
+    help="Output the tracebacks from the failing tasks. Only usable in conjunction with --errors.",
+    callback=validate_traceback_flag,
+)
+def status(network: str, errors: bool, with_traceback: bool):
     """
-    Get the status of the submitted network on alchemiscale.
+    Get the status of the submitted network on alchemiscale.\f
 
     Args:
         network: The name of the JSON file containing the FreeEnergyCalculationNetwork we should check the status of.
+        errors: Flag to show errors from the tasks.
+        with_traceback: Flag to show the complete traceback for the errored tasks.
 
     """
-    from .schema.fec import FreeEnergyCalculationNetwork
-    from .utils import AlchemiscaleHelper
+    from asapdiscovery.alchemy.schema.fec import FreeEnergyCalculationNetwork
+    from asapdiscovery.alchemy.utils import AlchemiscaleHelper
 
     # launch the helper which will try to login
     client = AlchemiscaleHelper()
     # load the network
     planned_network = FreeEnergyCalculationNetwork.from_file(network)
     # check the status
     client.network_status(planned_network=planned_network)
+    # Output errors
+    if errors:
+        task_errors = client.collect_errors(
+            planned_network, with_traceback=with_traceback
+        )
+        print(json.dumps(task_errors, indent=4))
```

Comment:
Instead of this, would recommend some human-readable structured output, such as separated, multiline text for each `Task`, the exceptions/tracebacks for each `ProtocolDAGResult`, as the default. 

Can then do `click.echo` of JSON form as you are doing here if  a `--json` flag is given. That would be suitable for redirection into a file from STDOUT.